app/subscriber.py

```diff
@@ -90,6 +90,8 @@ def offline_receipt_to_case(message: Message):
     if not payload:
         return  # Failed validation
 
+    unreceipt = payload.get('unreceipt', False)
```

Comment:
why are we defaulting this and not insisting that it's always sent?

Reply:
It should be able to just be passed through there. Not too sure why it's been done like that